core_backend/app/llm_call/llm_prompts.py

```diff
@@ -140,28 +142,31 @@ def get_prompt(cls) -> str:
         """
 
         return textwrap.dedent(
-            """
-            You are a labelling agent. You declare whether a message sent to an
-            {service_identity} is relevant to the topic or not. You classify each
-            message as one of """
-            + f"{cls.get_available_labels()}."
-            + """
+            f"""
+            You are a labelling agent. You declare whether a query sent to an
+            {SERVICE_IDENTITY} can be reasonable answered or not.
+            When it is answerable, you label it as {cls.ON_TOPIC.value}.
+            When it is not, you label it as {cls.OFF_TOPIC.value}.
+
```

Comment:
The examples here are specific to air quality and might trip this rail up if we change the purpose to something else I think... We should also read the examples from file perhaps?

Reply:
Yup I realized that lol I forgot to push my fix